tools/docs/github-query-items.ts

```diff
@@ -47,6 +47,18 @@ export interface Items {
 }
 
 export async function getOpenGitHubItems(): Promise<RenovateOpenItems> {
+  const result: RenovateOpenItems = {
```

Comment:
The first letter of the `RenovateOpenItems` name is _lowercased_ in other parts of the code.

Is it correct that the first letter is _Uppercased_ here?

Reply:
Yes, because it's a type and not a variable